### ISSUE TITLE
vscode-eslint blocks swc upgrade

### ISSUE THREAD

**Issue by alumni** [microsoft/vscode-eslint#1421]
Seems `vscode-eslint` is blocking `swc` from being upgraded.

Running `yarn install` after pulling changes from a remote (with changed lockfile) has the following error:
```
error An unexpected error occurred: "EPERM: operation not permitted, unlink '<PROJECT_ROOT>\\node_modules\\@swc\\core-win32-x64-msvc\\swc.win32-x64-msvc.node'".
```

```sh
yarn why @swc/core-win32-x64-msvc
=> Found "@swc/core-win32-x64-msvc@1.2.140"
info Reasons this module exists
   - "@nrwl#eslint-plugin-nx#@swc-node#register#@swc-node#core#@swc#core" depends on it
   - Hoisted from "@nrwl#eslint-plugin-nx#@swc-node#register#@swc-node#core#@swc#core#@swc#core-win32-x64-msvc"
```

In VSCode process explorer I can see `"<USER>\AppData\Local\Programs\Microsoft VS Code\Code.exe" --ms-enable-electron-run-as-node <USER>\.vscode\extensions\dbaeumer.vscode-eslint-2.2.2\server\out\eslintServer.js --node-ipc --clientProcessId=6736` has PID 17056.

Using SysInternals Process Explorer I can see that the process with PID 17056 has the following open handle: `<PROJECT_ROOT>\node_modules\@swc\core-win32-x64-msvc\swc.win32-x64-msvc.node`

Versions:
vscode: 1.64.2
vscode-eslint: 2.2.2

**Comment by dbaeumer** [comment]
I think that ESLint is configured in a way that it loads that module (`swc.win32-x64-msvc.node`). For performance reason the ESLint server keeps the ESLint module alive to avoid constant process spawning. This is by design. So there is little I can do to avoid this.

I will close the issue since I have no plans to implement a mode that supports process spawning.

**Comment by SamuelT-Beslogic** [comment]
What is the recommended solution? Is there anything plugin authors can do? (see https://github.com/unrs/unrs-resolver/issues/159 for example). Is there a change we need to request to the ESLint package? Is there a configuration that works around this issue ?
